### PR TITLE
Remove at() matcher from tests in federatedfilesharing

### PR DESCRIPTION
**Pull request by come-nc**
Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

File: apps/federatedfilesharing/tests/FederatedShareProviderTest.php [hunk 1]
```diff
@@ -548,11 +548,7 @@ public function testGetSharedBy() {
 		$node->method('getId')->willReturn(42);
 		$node->method('getName')->willReturn('myFile');
 
-		$this->addressHandler->expects($this->at(0))->method('splitUserRemote')
-			->willReturn(['user', 'server.com']);
-
-		$this->addressHandler->expects($this->at(1))->method('splitUserRemote')
-			->willReturn(['user2', 'server.com']);
+		$this->addressHandler->expects($this->never())->method('splitUserRemote');
```
Comment (nickvergessen) [inline apps/federatedfilesharing/tests/FederatedShareProviderTest.php:551]: This does not look reasonable but tests pass. Is the file not executed?

Comment (come-nc) [reply apps/federatedfilesharing/tests/FederatedShareProviderTest.php:551]: Putting once() or such in here made the test fail locally. Looking at it again I’m puzzled as to how this is possible 🤔

Maybe at() ensures the order but does not check the calls actually happen? (like no call is no failure, wrong order call is failure?)

Comment (nickvergessen) [reply apps/federatedfilesharing/tests/FederatedShareProviderTest.php:551]: > Maybe at() ensures the order but does not check the calls actually happen? (like no call is no failure, wrong order call is failure?)

Yep.... Might be exactly the reason why it is deprecated and being removed

**Review by nickvergessen — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**